.github/workflows/scripts/is-community-contributor.js

```diff
@@ -1,92 +1,49 @@
-			return context.payload[ type ].user;
-		}
-	}
-	return false;
-};
-
-const userIsBot = user => user.type && user.type === 'Bot';
-
-const userInOrg = async ( github, username, org ) => {
-	console.log( 'Checking for user %s in org %s', username, org );
-	try {
-		// First attempt to check memberships.
-		const result = await github.request( 'GET /orgs/{org}/memberships/{username}', {
-			org,
+// Note you'll need to install this dependency as part of your workflow.
+const { Octokit } = require('@octokit/action');
+
+// Note that this script assumes you set GITHUB_TOKEN in env, if you don't
+// this won't work.
+const octokit = new Octokit();
+
+const getPRAuthor = (payload) => {
+	return payload?.pull_request?.user?.login || null;
+}
+
+const isCommunityContributor = async (owner, repo, username)  => {
+	if (username) {
+		const {data: {permission}} = await octokit.rest.repos.getCollaboratorPermissionLevel({
+			owner,
+			repo,
 			username,
-		} );
-
-		if ( result && Math.floor( result.status / 100 ) === 2 ) {
-			console.log( 'User %s found in %s via membership check', username, org );
-			return true;
-		}
-	} catch ( error ) {
-		// A 404 status means the user is definitively not in the organization.
-		if ( error.status === 404 ) {
-			console.log( 'User %s NOT found in %s via membership check', username, org );
-			return false;
-		}
-
-		try {
-			// For anything else, we should also check public memberships.
-			const result = await github.request( 'GET /orgs/{org}/public_members/{username}', {
-				org,
-				username,
-			} );
-
-			if ( result && Math.floor( result.status / 100 ) === 2 ) {
-				console.log( 'User %s found in %s via public membership check', username, org );
-				return true;
-			}
-		} catch ( error ) {
-			if ( error.status === 404 ) {
-				console.log( 'User %s NOT found in %s via public membership check', username, org );
-				return false;
-			}
-		}
-	}
-	// If we've gotten to this point, status is unknown.
-	throw new Error( "Unable to determine user's membership in org" );
-};
-
-const userInNonCommunityOrgs = async ( orgs, github, username ) => {
-	let checked = 0;
-	for ( const org of orgs ) {
-		try {
-			const isUserInOrg = await userInOrg( github, username, org );
-			if ( isUserInOrg ) {
-				return true;
-			}
-			// If no error was thrown, increment checked.
-			checked++;
-		} catch( e ) {} // Do nothing.
+		});
+	
+		return permission === 'read' || permission === 'none';
 	}
 
-	if ( checked !== orgs.length ) {
-		throw new Error( "Unable to check user's membership in all orgs." );
+	return false;	
+}
+
+const addLabel = async(label, owner, repo, issueNumber) => {
+	await octokit.rest.issues.addLabels({
+		owner,
+		repo,
+		issue_number: issueNumber,
+		labels: [label],
+	});
+}
+
+const applyLabelToCommunityContributor = async () => {
+	const eventPayload = require(process.env.GITHUB_EVENT_PATH);
+	const username = getPRAuthor(eventPayload);
```

Comment:
```suggestion
	const username = getIssueAuthor(eventPayload);
```

Updated given the renamed function above